django/contrib/admindocs/views.py

```diff
@@ -261,11 +262,17 @@ def get_context_data(self, **kwargs):
                 verbose = verbose and (
                     utils.parse_rst(cleandoc(verbose), 'model', _('model:') + opts.model_name)
                 )
-                # Show properties and methods without arguments as fields.
+                # Show properties and cached_properties and methods without arguments as fields.
                 # Otherwise, show as a 'method with arguments'.
                 if isinstance(func, property):
                     fields.append({
-                        'name': func_name,
+                        'name': '%s (property)' % func_name,
+                        'data_type': get_return_data_type(func_name),
+                        'verbose': verbose or ''
+                    })
+                elif isinstance(func, cached_property):
+                    fields.append({
+                        'name': '%s (cached property)' % func_name,
```

Comment:
I don't see much value in added prefixes: `(cached property)`, `(property)`, and putting them in the `name` column can be misleading. We don't include prefixes for `classmethods`, `staticmethods` etc. 

If it's important for someone they can always add such information to the docstring.

Reply:
Agree, removing those.